src/main/java/com/epam/page/object/generator/utils/StringUtils.java

```diff
@@ -39,13 +38,13 @@ public static String splitCamelCase(String camel) {
 
     public static String firstLetterUp(String text) {
         return text.length() > 0
-            ? (text.charAt(0) + "").toUpperCase() + text.substring(1)
+            ? Character.toUpperCase(text.charAt(0)) + text.substring(1)
```

Comment:
This method is redundant. Use StringUtils#capitalize from apache commons

Reply:
Done